[2023-08-22 17:17:47,189] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: entregable3.extract_and_transform_data manual__2023-08-22T14:53:06.238231+00:00 [queued]>
[2023-08-22 17:17:47,232] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: entregable3.extract_and_transform_data manual__2023-08-22T14:53:06.238231+00:00 [queued]>
[2023-08-22 17:17:47,251] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-08-22 17:17:47,257] {taskinstance.py:1377} INFO - Starting attempt 2 of 6
[2023-08-22 17:17:47,258] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-08-22 17:17:47,449] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extract_and_transform_data> on 2023-08-22 14:53:06.238231+00:00
[2023-08-22 17:17:47,466] {standard_task_runner.py:52} INFO - Started process 14530 to run task
[2023-08-22 17:17:47,502] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'entregable3', 'extract_and_transform_data', 'manual__2023-08-22T14:53:06.238231+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/entregable.py', '--cfg-path', '/tmp/tmpm6dfkbjv', '--error-file', '/tmp/tmpxav7chvi']
[2023-08-22 17:17:47,507] {standard_task_runner.py:80} INFO - Job 56: Subtask extract_and_transform_data
[2023-08-22 17:17:48,002] {task_command.py:371} INFO - Running <TaskInstance: entregable3.extract_and_transform_data manual__2023-08-22T14:53:06.238231+00:00 [running]> on host eebeded14cb4
[2023-08-22 17:17:48,266] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Yamil
AIRFLOW_CTX_DAG_ID=entregable3
AIRFLOW_CTX_TASK_ID=extract_and_transform_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-22T14:53:06.238231+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-22T14:53:06.238231+00:00
[2023-08-22 17:17:53,018] {python.py:173} INFO - Done. Returned value was: None
[2023-08-22 17:17:53,055] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=entregable3, task_id=extract_and_transform_data, execution_date=20230822T145306, start_date=20230822T171747, end_date=20230822T171753
[2023-08-22 17:17:53,186] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-22 17:17:53,353] {dagrun.py:1081} ERROR - Failed to get task '<TaskInstance: entregable3.transform_data manual__2023-08-22T14:53:06.238231+00:00 [removed]>' for dag 'entregable3'. Marking it as removed.
[2023-08-22 17:17:53,354] {dagrun.py:1081} ERROR - Failed to get task '<TaskInstance: entregable3.get_data_from_api manual__2023-08-22T14:53:06.238231+00:00 [removed]>' for dag 'entregable3'. Marking it as removed.
[2023-08-22 17:17:53,394] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
